system-tests/services/_scripts/external-volumes/volclean.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
```

Comment:
Your concern is valid @weblancaster. I think we can just merge it as it is right now and revisit it later in the future. I will need to speak to @lloesche in order for him to make changes on his side. For now, I will add a TODO comment to the script.